src/components/sidebar/Sidebar.css

```diff
@@ -0,0 +1,45 @@
+.sidebar{
+    height:  calc(100vh - 4rem);
+    width: 16rem;
+}
+.sidebarlist{
+    height: auto;
+    width: 100%;
+}
+.sidebarlist .row {
+    width: 100%;
+    height: 3.5rem;
+    font-size: 1.5rem;
+    color: inherit;
+}
+.row .side-icon {
+    flex: 30%;
+    display: grid;
+    place-items: center;
+}
+.row .side-title {
+    flex: 70%;
+    display: block;
+  
+}
+.icon-div{
+    border-radius: 50%; 
+    width:3.5rem;
+    height:3.5rem;
+}
+/* hover styles */
+.item-icon:hover{
+    background-color: #666161;
+}
+.item-title:hover {
+    cursor: pointer;
+    /* border-radius: 10px; */
```

Comment:
If not needed you can remove the commented code

Reply:
sure .